src/telemetry/collectors.cpp

```diff
@@ -80,7 +80,7 @@ const nlohmann::json GetResourceUsage(std::filesystem::path root_directory) {
   cpu["usage"] = cpu_total.second;
   const auto vm_max_map_count = utils::GetVmMaxMapCount();
   return {{"cpu", cpu},
-          {"memory", utils::GetMemoryUsage()},
+          {"memory", utils::GetMemoryRES()},
```

Comment:
When did this happen?

Reply:
Here: https://github.com/memgraph/memgraph/pull/1426